I'm runnin into an error where 'python manage.py test' is not running the tests found in /trailer_bookings/tests.py
